src/components/study/job-approval-status.tsx

```diff
@@ -41,14 +42,26 @@ export const CodeApprovalStatus: FC<{ job: LatestJobForStudy }> = ({ job }) => {
     return <JobApprovalStatus statusChange={codeStatusChange} />
 }
 
-export const FileApprovalStatus: FC<{ job: LatestJobForStudy }> = ({ job }) => {
+export const FileApprovalStatus: FC<{ job: LatestJobForStudy; orgSlug: string }> = ({ job, orgSlug }) => {
     const filesStatusChange = job.statusChanges.find((statusChange) => {
-        return statusChange.status === 'FILES-APPROVED' || statusChange.status === 'FILES-REJECTED'
+        return (
+            statusChange.status === 'FILES-APPROVED' ||
+            statusChange.status === 'FILES-REJECTED' ||
+            statusChange.status === 'JOB-ERRORED'
```

Comment:
could DRY it a bit:

```suggestion
            ['FILES-APPROVED', 'FILES-REJECTED' , 'JOB-ERRORED'].includes(statusChange.status) 
```

Reply:
actually there's a FINAL_STATUS const in dashboard/table that includes this as well as CODE-REJECTED.  I wonder if this should have CODE-REJECTED as well?  If so lets move that const to lib/types as JOB-FINAL-STATUSES or something and use it in both places